qa/workunits/cephtool/test.sh

```diff
@@ -551,7 +551,7 @@ function test_tiering_9()
 
 function test_auth()
 {
-  ceph auth add client.xx mon allow osd "allow *"
+  ceph auth add client.xx mon 'allow *' osd "allow *"
```

Comment:
why single quotes for one and double for the other? trying to exercise both in one go?

Reply:
probably they are not on purpose. i don't think they are different from ceph cli's point of view. the quotes are only visible to the shell.